app/src/main/res/values/strings.xml

```diff
@@ -74,4 +74,7 @@
 
     <string name="log_your_own_outfit">Log your own outfit</string>
 
+    <string name="invalid_manual_outfit_log">Cannot have all clothes selected to nothing!</string>
```

Comment:
Can we change the toast message to something like `Please select at least 1 item of clothing` ?

Reply:
Sounds good